app/views/publish/providers/accredited_providers/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :page_title, title_with_error_prefix("About the accredited provider - Add accredited provider - #{@provider.name_and_code}", @accredited_provider_form.errors.present?) %>
+
+ <%= form_with(
+        model: @accredited_provider_form,
+        url: publish_provider_recruitment_cycle_accredited_providers_path,
+        method: :put
+      ) do |f| %>
+
+  <%= f.govuk_error_summary %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+  <% content_for :before_content do %>
+    <%= govuk_back_link_to(publish_provider_recruitment_cycle_accredited_providers_path) %>
+  <% end %>
+
+        <%= f.govuk_text_area(
+          :description,
+          label: { text: t(".title"), size: "l", tag: "h1" },
+          hint: { text: t(".hint") },
+          caption: { text: t(".caption", provider_name: @recruitment_cycle.providers.find_by(provider_code: params[:accredited_provider_code]).provider_name), size: "l" },
```

Comment:
```suggestion
          caption: { text: t(".caption",provider_name: @accredited_provider.provider_name), size: "l" },
```

Move this to controller please
```
@recruitment_cycle.providers.find_by(provider_code: params[:accredited_provider_code])
```

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3573/commits/b331083a53b2e9083956e5419f17e3bff3242fee